.github/workflows/canister-tests.yml

```diff
@@ -955,8 +955,23 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/setup-didc
+      - name: 'Get latest release'
```

Comment:
Does this needs to be done every time we introduce a breaking change?

Do you have a script or how do you remember how to do it?

Reply:
I usually pull it out again from git history whenever I need it. My hope was that it would not be worth it improving the infrastructure to do that, as we would eventually stop making breaking changes (as APIv2 would be finished)...